user-guide/Reference/faq/General_configuration.md

```diff
@@ -80,7 +89,7 @@ To turn a DMA into a time client, do the following.
 
 1. Log on locally to the DMA, or establish a remote desktop session with it.
 
-1. Open the *Services* dialog box (*Start \> Settings \> Control Panel \> Administrative Tools \> Services*), and select the *Windows Time* service.
+1. Open the *Services* dialog box Start \> Settings \> Control Panel \> Administrative Tools \> Services, and select the *Windows Time* service.
```

Comment:
In this case, the parentheses were necessary, as the content of the parentheses was not part of the grammatical structure of the sentence. If you want to leave out the parentheses, you need to adjust the sentence. For example: 
"Open the *Services* dialog box via *Start* \> *Settings* \> *Control Panel* \> *Administrative Tools* \> *Services*, ..."